script/bcar.js

```diff
@@ -2977,8 +2977,13 @@ CommandCombine([
         Action: args => {
             if (CurrentScreen == "ChatRoom") {

```

Comment:
For future reference: might want to consider `ChatRoomClearAllElements()` here instead of manually copying parts of its logic.